java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/travel/WikivoyageLangPreparation.java

```diff
@@ -107,8 +108,9 @@ public String getType() {
 			return type;
 		}
 	}
-	
-	public static void main(String[] args) throws IOException, ParserConfigurationException, SAXException, SQLException {
+
+	public static void main(String[] args) throws IOException, ParserConfigurationException, SAXException, SQLException,
+			XmlPullParserException, InterruptedException {
```

Comment:
Add forth argument 
_osm_wiki_folder_
and 
_osm_wiki_file_prefix_ (what file to use)

In MainUtilities add help to function to _printSynopsys_